test/nodes/test_search_hybrid.py

```
@@ -0,0 +1,741 @@
+
+# ---------------------------------------------------------------------------
+# Install rank_bm25 stub before loading hybrid_search
+# ---------------------------------------------------------------------------
+_STUB_MODULES = [
+    'rank_bm25',
+]
+
+_original_modules = {}
+
+
+def _install_bm25_stub():
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_install_bm25_stub`

Add return type annotation: `None`

(ANN202)